src/app/radio/radio-index/radio-index.component.html

```diff
@@ -0,0 +1,55 @@
+<h1>Radio Button</h1>
+<h5>Il componente Radio Button utilizzabile in un form</h5>
+
+<ul class="nav nav-tabs" id="radio-tab" role="tablist">
+  <li class="nav-item">
+    <a class="nav-link active" id="radio-description-tab" data-toggle="tab" href="#radio-description-tab-content" role="tab"
+      aria-controls="radio-description-tab-content" aria-selected="true">Descrizione</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" id="radio-api-tab" data-toggle="tab" href="#radio-api-tab-content" role="tab" aria-controls="radio-api-tab-content"
+      aria-selected="false">API</a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link" id="radio-examples-tab" data-toggle="tab" href="#radio-examples-tab-content" role="tab" aria-controls="radio-examples-tab-content"
+      aria-selected="false">Esempi</a>
+  </li>
+</ul>
+<div class="tab-content" id="radio-content-tab">
+  <div class="tab-pane p-3 fade show active" id="radio-description-tab-content" role="tabpanel" aria-labelledby="radio-description-tab-content">
+    <div [innerHTML]="component.description"></div>
+  </div>
+  <div class="tab-pane p-3 fade" id="radio-api-tab-content" role="tabpanel" aria-labelledby="radio-api-tab-content">
+    <!-- TODO: da spostare in una componente ad hoc -->
```

Comment:
In questo file ci sono un paio di TODO to do. 😄 
Nel caso questa lavorazione non sia parte di questo task, che ne dite di creare una issue separata?

Reply:
Concordo nella creazione di un task a parte